src/libopensc/types.h

```diff
@@ -225,6 +225,11 @@ typedef struct sc_acl_entry {
 #define SC_FILE_EF_CYCLIC		0x06
 #define SC_FILE_EF_CYCLIC_TLV		0x07
 
+/* Data compression */
+#define SC_FILE_COMPRESSED_AUTO		0x08
+#define SC_FILE_COMPRESSED_ZLIB		0x09
+#define SC_FILE_COMPRESSED_GZIP		0x0a
```

Comment:
I would  probably prefer to have them in separate "namespace" like `SC_FILE_FLAG_*`. Given that you set them as a bitfiled, they should not be overlapping values.